src/install.rst

```diff
@@ -213,7 +213,8 @@ Set up a Nextstrain runtime
 
       The ambient runtime does not require a particular setup method; it will work as long as the programs you wish to use are available.
 
-      The following describes how to accomplish this using a custom Conda environment as an example. You should be familiar with the `basics of Conda <https://conda.io/projects/conda/en/latest/user-guide/getting-started.html>`__ before proceeding.
+      .. Suggest Python 3.10.13 because Augur on Bioconda won't resolve dependencies on Python ≥3.11: <https://github.com/nextstrain/augur/issues/1334>
+      The following describes how to accomplish this by creating a new custom Conda environment, as an example. You should be familiar with the `basics of Conda <https://conda.io/projects/conda/en/latest/user-guide/getting-started.html>`__ before proceeding. If you want to add Nextstrain to an existing Conda environment, please make sure you're using Python ≤3.10 and activate that environment instead of creating a new one.
```

Comment:
Suggestion: add a newline between the comment and displayed text. It seems to build fine, but is less readable and my text editor's syntax highlighting thinks the displayed text is part of the comment.

Reply:
done.